### PR TITLE
users are now reminded to close their manual applicant sources when closing a job opening - closes #136

### LINKED ISSUE #136: Add some sort of closeout process to ATS that shows manual applicant sources
We need to help remind employers to close campaign sources that are "manual".

### PR DESCRIPTION
**Pull request by petersonbrock**
screen shot 2018-05-22 at 8 55 10 am

### CODE REVIEW

**Comment by James1x0** [comment]
@petersonbrock could you add a screenshot to this PR?

File: app/components/modals/ats-close-job-opening.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+  createConfirm () {
+    this.setProperties({
+      responded: false,
+      _originalArgs: arguments
+    });
+
+    $('#' + this.get('modalId')).modal({
+      detachable: true,
+      onHidden: () => {
```
Comment (James1x0) [inline app/components/modals/ats-close-job-opening.js]: Add `closable: true` here.

File: app/templates/components/modals/ats-close-job-opening.hbs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{{#if hasBlock}}
+  {{yield startConfirmation}}
+{{/if}}
+
+{{#ui-modal name="profile" id=modalId}}
+  <i class="close icon"></i>
```
Comment (James1x0) [inline app/templates/components/modals/ats-close-job-opening.hbs]: Remove the close icon

File: app/components/modals/ats-close-job-opening.js (hunk 1 above)
Comment (petersonbrock) [reply app/components/modals/ats-close-job-opening.js]: you mean false?

Comment (James1x0) [reply app/components/modals/ats-close-job-opening.js]: 😢 Yes

**Pull request merged**